docs/streams/developer-guide/config-streams.html

```diff
@@ -1017,6 +1022,20 @@ <h4><a class="toc-backref" href="#id14">state.dir</a><a class="headerlink" href=
           this path must be unique for each such instance.</div>
       </blockquote>
     </div>
+    <div class="section" id="task-assignor-class">
+      <h4><a class="toc-backref" href="#id39">task.assignor.class</a><a class="headerlink" href="#task-assignor-class" title="Permalink to this headline"></a></h4>
+      <blockquote>
+        <div>A task assignor class or class name implementing the
+          <code>org.apache.kafka.streams.processor.assignment.TaskAssignor</code> interface. Defaults to the
+          high-availability task assignor. One possible alternative implementation provided in Apache Kafka is
+          <code>org.apache.kafka.streams.processor.assignment.StickyTaskAssignor</code>, which was the default task
+          assignor before KIP-441 and prioritizes stickiness over task availability. Alternative implementations of
+          the task assignment algorithm can be provided in the classpath of the Kafka Streams application. This
+          configuration will then be set to the fully qualified class name of the desired
+          <code>TaskAssignor</code> implementation.
```

Comment:
maybe this is just me, but the wording here feels a bit confusing. Here's one suggestion for clearing this up (feel free to word it differently):

```suggestion
          the task assignment algorithm can be plugged into the application by implementing a custom <code>TaskAssignor</code> and setting this config to the name of the custom task assignor class.  
```